pkg/protocol/protobuf/vcsec.proto

```diff
@@ -225,9 +225,63 @@ message UnsignedMessage {
     }
 }
 
+enum ClosureState_E {
+    CLOSURESTATE_CLOSED = 0;
+    CLOSURESTATE_OPEN = 1;
+    CLOSURESTATE_AJAR = 2;
+    CLOSURESTATE_UNKNOWN = 3;
+    CLOSURESTATE_FAILED_UNLATCH = 4;
+    CLOSURESTATE_OPENING = 5;
+    CLOSURESTATE_CLOSING = 6;
+}
+
+enum VehicleLockState_E {
+    VEHICLELOCKSTATE_UNLOCKED = 0;
+    VEHICLELOCKSTATE_LOCKED = 1;
+    VEHICLELOCKSTATE_INTERNAL_LOCKED = 2;
+    VEHICLELOCKSTATE_SELECTIVE_UNLOCKED = 3;
+}
+
+message ClosureStatuses
+{
+    ClosureState_E   frontDriverDoor = 1;
```

Comment:
nit: can we make `0` as UNKNOWN?

Reply:
This isn't an enum type; those are field numbers, not enum values. :)